apps/openssl.c

```diff
@@ -207,6 +207,11 @@ int main(int argc, char *argv[])
     if (p != NULL && strcmp(p, "on") == 0)
         OSSL_debug_set(OSSL_DEBUG_PKCS5V2, internal_debug_cb, bio_err);
 
+    /* set this line to get debugging info about PKCS#12 key generation */
+    p = getenv("OPENSSL_DEBUG_PKCS12_KEYGEN");
+    if (p != NULL && strcmp(p, "on") == 0)
+        OSSL_debug_set(OSSL_DEBUG_PKCS12_KEYGEN, internal_debug_cb, bio_err);
+
```

Comment:
An idea comes to my mind: why not have a standard way of enabling debug features via environment? Something like
		
```C
    if (OSSL_debug_enabled("OPENSSL_DEBUG_PKCS12_KEYGEN")
        OSSL_debug_set(OSSL_DEBUG_PKCS12_KEYGEN, internal_debug_cb, bio_err);
```

The need for using double quotes could be even circumvented by implementing `OSSL_debug_enabled()` as a macro using the  stringification operator:

```
#define OSSL_debug_enabled(var) OSSL_check_debug_enabled(#var)
```

The `OSSL_check_debug_enabled()` function could be flexible in what it accepts as value for the environment value, such that the user does not have to remember the precise value. Anything like `on|yes|true|enabled` would be accepted:

```
OPENSSL_DEBUG_PKCS12_KEYGEN=on
OPENSSL_DEBUG_PKCS12_KEYGEN=yes
OPENSSL_DEBUG_PKCS12_KEYGEN=true
OPENSSL_DEBUG_PKCS12_KEYGEN=enabled
```

Reply:
Or, alternatively, just check for `OSSL_debug_enabled(PKCS12_KEYGEN)`, where

```
#define OSSL_debug_enabled(var) OSSL_check_debug_enabled("OPENSSL_DEBUG" #var)
```